domain/dailydebitlimit.go

```diff
@@ -14,8 +14,9 @@ const maximumDailyDebitLimit = 900000
 // dailyDebitLimit is the aggregate root for an account daily debit limit
 // policy.
 type dailyDebitLimit struct {
-	// UsedAmount is the sum of debit amounts used during the period, in cents.
-	UsedAmount int64
+	// ConsumedAmount is the sum of debit amounts consumed during the period, in
```

Comment:
"during the period" sounds like a certain period is implied, perhaps the period should be mentioned? E.g. `ConsumedAmount is the sum of debit amounts consumed during the daily period...`.  If there is no need to emphasise any particular period, perhaps it should sound like `ConsumedAmount is the sum of debit amounts consumed during a certain period...`

Also, I think you meant to use the amount in singular: 

```suggestion
	// ConsumedAmount is the sum of debit amount consumed during the period, in
```

Reply:
Sorry I did not consider the top comment for the `dailyDebitLimit ` struct which mentions the daily limit and hence the daily period, you can disregard my first paragraph in the comment.